File: 2st/Yeonwoo/간단한 식 계산하기.py
def solution(binomial):
    answer = 0
    #a = 0
    #b = 0
    #op = ''
    a,op,b = binomial.split()
    if op == '+':
        answer = int(a)+int(b)
    elif op == '-':
        answer = int(a)-int(b)
    elif op == '*':
        answer = int(a)*int(b)        
    return answer

File: 3st/Juram/programmer/간단한 식 계산하기.js
function solution(binomial) {
    var answer = 0;
    var bi = binomial.split(' ');
    switch(bi[1]){
        case "+": 
            answer = Number(bi[0])+Number(bi[2]);
            break;
        case "-": 
            answer = Number(bi[0])-Number(bi[2]);
            break;
        case "*": 
            answer = Number(bi[0])*Number(bi[2]);
            break;
    }
    return answer;
    
}